server/views/pages/statement/your-statement.html

```diff
@@ -55,69 +54,49 @@ <h1 class="govuk-heading-xl mainHeading">Your use of force statement</h1>
   </div>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0 govuk-label--xs statement" data-qa="statement">{{ data.statement }}  </p>
+      <p class="govuk-!-margin-bottom-1 govuk-!-margin-top-0 govuk-label--xs statement" data-qa="statement">
+        {{ data.statement }} </p>
```

Comment:
think this is going to add extra padding to the start of the statement as it has `white-space: pre-wrap` - needs to be on one line

Reply:
Yes, that happened when I used the linter. 
Amended